x/metadata/types/scope_test.go

```diff
@@ -36,90 +36,110 @@ func (s *ScopeTestSuite) SetupSuite() {
 }
 
 func (s *ScopeTestSuite) TestScopeValidateBasic() {
+	ns := func(scopeID, scopeSpecification MetadataAddress, owners []Party, dataAccess []string, valueOwner string) *Scope {
```

Comment:
I wanted the name to be super short like this, and I didn't feel comfortable adding `ns` to the `types` package, even if only for unit tests. By defining it again like that (where multiple tests can use it), it's no different than just using `NewScope`; when it's updated, a lot of things need updating in the exact same way so that the tests can continue to operate as they previously did. By defining it individually for each test that needs it, it limits the amount of changes needed when the scope changes. If a test starts failing, the `ns` function can be updated in just that test as needed, and the other tests (that are still passing) don't require attention.